spec/lib/web_of_science/record_spec.rb

```diff
@@ -1,23 +1,30 @@
 require 'htmlentities'
 
 describe WebOfScience::Record do
-  let(:encoded_record) { File.read('spec/fixtures/wos_client/wos_encoded_record.html') }
-  let(:decoded_record) do
+  let(:wos_encoded_record) { File.read('spec/fixtures/wos_client/wos_encoded_record.html') }
+  let(:wos_decoded_record) do
     coder = HTMLEntities.new
-    coder.decode(encoded_record)
+    coder.decode(wos_encoded_record)
   end
-  let(:wos_record_encoded) { described_class.new(encoded_record: encoded_record) }
-  let(:wos_record_decoded) { described_class.new(record: decoded_record) }
+  let(:wos_record_encoded) { described_class.new(encoded_record: wos_encoded_record) }
+  let(:wos_record_decoded) { described_class.new(record: wos_decoded_record) }
   let(:wos_uid) { 'WOS:A1972N549400003' }
 
+  let(:medline_encoded_record) { File.read('spec/fixtures/wos_client/medline_encoded_record.html') }
+  let(:medline_decoded_record) do
+    coder = HTMLEntities.new
+    coder.decode(medline_encoded_record)
+  end
+  let(:medline_record_encoded) { described_class.new(encoded_record: medline_encoded_record) }
+  let(:medline_record_decoded) { described_class.new(record: medline_decoded_record) }
+  let(:medline_uid) { 'MEDLINE:24452614' }
```

Comment:
I don't see where these two statements are used.

Reply:
That's right, they might not be used yet.  They will be useful for additional specs on MEDLINE records, e.g. extracting things for PubHash mapping.  Rather than add them later, they were added while including the MEDLINE fixture data.  Since they are `let` statements, they are no-op until they are used; and I expect they will be used.  If I'm wrong about that, I guess they could be removed soon enough.

Reply:
I will add specs to use these fixture data.